### ISSUE TITLE
fix: Could not resolve all artifacts for configuration ':app:productionReleaseRuntimeClasspath'

### ISSUE THREAD

**Issue by graphicbeacon** [VeryGoodOpenSource/very_good_cli#104]
**Description**
Hello fellow devs, thank you for putting this tool together. It helped me get started on my current client project. However I'm getting the following error when building this project:

```bash
Note: /Users/joppong/tools/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_secure_storage-4.0.0/android/src/main/java/com/it_nomads/fluttersecurestorage/FlutterSecureStoragePlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.                    
Note: /Users/joppong/tools/flutter/.pub-cache/hosted/pub.dartlang.org/local_auth-1.1.0/android/src/main/java/io/flutter/plugins/localauth/LocalAuthPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.                    
                                                                        
FAILURE: Build failed with an exception.                                
                                                                        
* What went wrong:                                                      
Execution failed for task ':app:lintVitalDevelopmentRelease'.           
> Could not resolve all artifacts for configuration ':app:productionReleaseRuntimeClasspath'.
   > Failed to transform libs.jar to match attributes {artifactType=processed-jar, org.gradle.libraryelements=jar, org.gradle.usage=java-runtime}.
      > Execution failed for JetifyTransform: /Users/joppong/Projects/clients/sumgroup_app/packages/sumgroup_flutter/build/app/intermediates/flutter/productionRelease/libs.jar.
         > Transform's input file does not exist: /Users/joppong/Projects/clients/sumgroup_app/packages/sumgroup_flutter/build/app/intermediates/flutter/productionRelease/libs.jar. (See https://issuetracker.google.com/issues/158753935)
                                                                        
* Try:                                                                  
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
                                                                        
* Get more help at https://help.gradle.org                              
                                                                        
BUILD FAILED in 34s                                                     
Running Gradle task 'assembleRelease'...                                
Running Gradle task 'assembleRelease'... Done                      35.1s
Gradle task assembleRelease failed with exit code 1
```

**Steps To Reproduce**
1. Run the command `flutter build apk --target lib/main_production.dart`
2. See error

**Expected Behavior**
The build should be successful. 

**Additional Context**
I noticed the warnings with **flutter_secure_storage** and **local_auth** so I went ahead and bootstrapped a `flutter create .` project with both packages. Running the `flutter build` command still displays the warning but then the build passes.

**Comment by graphicbeacon** [comment]
@felangel Thanks for investigating this one Felix :)

**Comment by graphicbeacon** [comment]
@felangel Hey, found anything so far or is there any workaround I could use?

**Comment by pepealonso95** [comment]
@graphicbeacon Quick workaround, comment out flavors in app/build.gradle (as long as you haven't been using them)

**Comment by graphicbeacon** [comment]
@pepealonso95 Thanks for the workaround. Hoping we can get a solution soon :)

**Comment by binoypatel** [comment]
I tried the following work around and it worked well so far:
[https://github.com/flutter/flutter/issues/58247#issuecomment-636253593](https://github.com/flutter/flutter/issues/58247#issuecomment-636253593)

**Comment by felangel** [comment]
Hey everyone sorry for the delayed response! I’m planning to address this within the next day or so. Thanks for your patience 🙏

**Comment by hmhung2531991** [comment]
@felangel Did you have any update?

**Comment by felangel** [comment]
Fix should be out today. The fix is currently in review and we will publish a new version as soon as they are approved. Thanks again for everyone's patience! 👍

**Comment by felangel** [comment]
This should be resolved in [v0.1.7](https://pub.dev/packages/very_good_cli) 🎉

**Comment by erlangparasu** [comment]
@felangel how to build apk with --profile? --release?
